src/lsm/manifest_level.zig

```diff
@@ -117,16 +117,34 @@ pub fn ManifestLevelType(
             level.table_count_visible -= 1;
         }
 
+        /// Remove the given table from the level assuming it's visible to `lsm.snapshot_latest`.
+        /// Returns the same, unmodified table passed in to differentiate itself from 
+        /// remove_table_invisible and guard against using the wrong function.
```

Comment:
Great comment here, and good idea to differentiate function signatures for safety.